parlai/core/torch_generator_agent.py

```diff
@@ -628,6 +630,7 @@ def _init_cuda_buffer(self, batchsize, maxlen, force=False):
 
         This is also used in distributed mode to force a worker to sync with others.
         """
+        return
```

Comment:
Hmm I assume this is a line added for debugging?

Reply:
speaking of, can we possibly make this a flag to turn off, i.e. once we make this `debug` mode? i really would appreciate being able to turn this off when debugging...

Reply:
On the roadmap to get rid of it entirely